### DISCUSSION TITLE
ポッドキャストの最新エピソードを画面上のどこかに載せる

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#279] [要望・アイデア・不具合]
「[エンジニアストーリー by Qiita](https://pitpa.jp/playlist/engineerstory)」が始まったので、画面上のどこか（右のサイドバーとか？）にリンクやバナーを置いて、ユーザーに気付いてもらいやすくするといいかな〜と思いました。

<img width="600" alt="Screen Shot 2022-08-17 at 18 13 00" src="https://user-images.githubusercontent.com/1148320/185082161-6a087e2e-428a-4858-9054-9cc8f49c8383.png">

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ありがとうございます！🙏
早速、運営からのお知らせ欄に掲載いたしました！✨
いつもありがとうございます🙇

![image](https://user-images.githubusercontent.com/103916185/185091627-04cd0977-604d-4595-b7cf-f94bf04edafd.png)

**Reply by JunichiIto** [reply]
こちらこそ迅速な対応ありがとうございます😄
